app/assets/javascripts/application.js

```diff
@@ -31,7 +31,6 @@
 //= require scp-igv
 //= require scp-ideogram
 //= require scp-dot-plot
-//= require react_rails/bulkDownloadClipboard
```

Comment:
Don't we actually want this file in there?

Reply:
We did, but no longer.

The use of `window.SCP` and jQuery caused errors in the Jest tests, so I reverted my de-duplicating efforts in `react_rails/bulkDownloadClipboard` to enable Jest unit tests for the React components in `DownloadButton.js`.